cmd/dump.go

```diff
@@ -107,6 +107,14 @@ configure Kong.`,
 			return nil
 		}
 
+		ok, err := confirmFileOverwrite(dumpCmdKongStateFile, dumpCmdStateFormat, assumeYes)
+		if err != nil {
+			return err
+		}
+		if !ok {
+			return nil
+		}
```

Comment:
```suggestion
		if ok, err := confirmFileOverwrite(dumpCmdKongStateFile, dumpCmdStateFormat, assumeYes); err != nil {
			return err
		} else if !ok {
			return nil
		}
```

nit - the variables don't need to be function-scoped

Also another nit: `ok` is a name in Go that has a well-defined meaning (meaning that a type assertion or key lookup succeeded). How about changing this to `hasConsent`?

Also applies to other repetitions.